dataset-snippets/measurement_by_age_and_gender.plotnine

```diff
@@ -0,0 +1,33 @@
+# Use snippet 'measurement_by_age_and_gender' to plot joined demographics and measurements dataframes.
+# This plot assumes 'measurement_df' was created using snippet 'Basic operations -> join_dataframes' to
+# join together demographics and measurements dataframes.
+# See also https://plotnine.readthedocs.io/en/stable/
+
+
+# There could be many different measurements in the dataframe. By default, plot the first one.
+measurement_to_plot = measurement_df.STANDARD_CONCEPT_NAME.unique()[0]
+
+# Create a derived variable for age group.
+measurement_df['age'] = (pd.to_datetime('today').year
+                                     - measurement_df.DATE_OF_BIRTH.map(lambda x: x.year))
+measurement_df['age_group'] = pd.cut(measurement_df['age'],
+                                                 [-np.inf, 34.5, 49.5, 64.5, np.inf],
+                                                 labels=["<35", "35-49", "50-64", "65+"])
+
+# meas_filter is a column of True and False
+meas_filter = ((measurement_df.STANDARD_CONCEPT_NAME == measurement_to_plot)
+  & (measurement_df.UNIT_CONCEPT_NAME != 'No matching concept') 
+  & (measurement_df.GENDER != 'No matching concept') 
+  & (measurement_df.VALUE_AS_NUMBER < 9999999)  # Get rid of nonsensical outliers.
+)
+
+(ggplot(measurement_df[meas_filter], aes(x = 'age_group', y = 'VALUE_AS_NUMBER')) +
+    geom_boxplot() +
+    stat_summary(fun_data = get_boxplot_fun_data, geom = 'text', size = 10,
+                 position = position_dodge(width = 0.9), va = 'top') +
+#    scale_y_log10() +  # Uncomment if the data looks skewed.
+    coord_flip() +
+    facet_wrap(('STANDARD_CONCEPT_NAME + UNIT_CONCEPT_NAME', 'GENDER'), ncol = 2, scales = 'free') +
```

Comment:
can you format the title here? I'm getting:

`Body heightInches` as my chart title

Reply:
I think that's a bug in plotnine, ggplot2 handles it fine. 

Let me tackle that in a follow up PR. I can try to work around using https://ggplot2.tidyverse.org/reference/labellers.html

Reply:
Filed https://github.com/all-of-us/workbench-snippets/issues/21